tests/platform/broadcom/test_ser.py

```diff
@@ -0,0 +1,85 @@
+import os
+import time
+import random
+import logging
+import pprint
+
+import pytest
+
+logger = logging.getLogger(__name__)
+
+pytestmark = [
+    pytest.mark.disable_loganalyzer,
+    pytest.mark.broadcom
+]
+
+BASE_DIR = os.path.dirname(os.path.realpath(__file__))
+DUT_TMP_DIR = os.path.join('tmp', os.path.basename(BASE_DIR))
+
+FILES_DIR = os.path.join(BASE_DIR, 'files')
+SER_INJECTOR_FILE = 'ser_injector.py'
+DUT_WORKING_DIR = '/tmp/'
+
+def disable_ssh_timout(dut):
```

Comment:
this should be more generic, not with broadcom platform.

Reply:
Ack